Pipfile

```diff
@@ -0,0 +1,26 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+apertium-streamparser = "*"
+chardet = "*"
+requests = "*"
+tornado = "*"
+commentjson = "*"
+
+[dev-packages]
+coverage = "*"
+coveralls = "*"
+flake8-commas = "*"
+flake8-quotes = "*"
+flake8 = "*"
+idna = "*"
```

Comment:
idna, mccabe, and typed-ast are transitive. I think pycodestyle and pyflakes are as well.